src/fragments/lib-v1/auth/js/user_session/fetch_auth_session.mdx

```diff
@@ -0,0 +1,11 @@
+```ts
+async function handleFetchAuthSession() {
```

Comment:
Nit on the function name, we're not really "handling" anything here like a UI event or form submission

```suggestion
import { fetchAuthSession } from 'aws-amplify/auth';

async function loadSessionTokens() {
```